src/index.js

```diff
@@ -22,6 +24,38 @@ const rederMovies = async (genresType) => {
         </div>
         </div>`;
   }
+
+  const commentButton = document.querySelectorAll('.comment-btn-div button');
+  const commentPopup = () => {
+    commentButton.forEach((button, index) => {
+      button.addEventListener('click', () => {
+        commentSection.classList.add('show');
+        bodyfix.classList.add('static');
+
+        commentSection.innerHTML = `<div class='comment-js'>
+        <div class='name-closeicon'>
+        <h2>${movies[index].name}</h2>
+        <i class="fa fa-times"></i>
+        </div>
+        <div class='comment-img'><img src='${movies[index].image.original}'></div>
+        <div class='summary'>${movies[index].summary}</div>
+        <ul>
+        <li>Genre: ${movies[index].genres.join(',')}</li>
+        <li>tvmaze: ${movies[index].url}</li>
```

Comment:
I think this one can be made like this <a href="${movies[index].url}">link to tvmaze</a>